src/views/dashboard/Pam/RiskSummary.vue

```diff
@@ -135,7 +135,6 @@ export default {
           show: true,
           min: 0,
           max: max,
-          interval: 1,
           position: 'top',
           axisLine: { show: false },
           axisTick: { show: false },
```

Comment:
There are no significant code differences to address based on the knowledge cutoff provided in 2021-09-01.

However, when looking at it from an optimization perspective:

To reduce reuslting in case of large datasets that doesn't contain negative values for example: `filteredData` has been modified and now only keeps rows where value is equal to zero (or more simply just returns all rows) 

```
if(data.some(item => item.value === 0)){
 filteredData = [...data.slice()] 
} 
    
return filtered Data;  
```